### DISCUSSION TITLE
How to use `setFieldsValue` for a `Select` field in a `Form`?

### DISCUSSION THREAD

**Discussion by daniellimzj** [ant-design/ant-design#31979] [Q&A]
Hi, I am trying to use `setFieldsValue` to set a form's fields based on React state. I noticed that for a `Switch` component in a form, it is necessary to set the `valuePropName="checked"` field for the form item, like 
```
<Form.Item label="Switch" name="Switch" valuePropName="checked">
    <Switch
        onChange={(e) => {
            setSwitch(e)
        }}
    />
</Form.Item> 
```

Is there a `valuePropName` I must use for a `Select` field? I am unable to use `setFieldsValue` to populate my Select field with a default value based off my React state.

**Comment by MJLfighting** [comment]
maybe you could try wrapping  'getFieldDecorator' around 'Form'
